features/attached_enable.feature

```diff
@@ -59,6 +59,16 @@ Feature: Enable command behaviour when attached to an UA subscription
             """
             This command must be run as root (try using sudo).
             """
+        And I verify that running `ua enable foobar --format json` `as non-root` exits `1`
+        And stdout is formatted as `json` and has keys:
+            """
+            result processed_services failed_services needs_reboot
+            errors warnings
+            """
+        And I will see the following on stdout:
+            """
+            {"errors": [{"message": "This command must be run as root (try using sudo).", "service": null, "type": "system"}], "failed_services": [], "needs_reboot": false, "processed_services": [], "result": "failure", "warnings": []}
```

Comment:
@lucasmoura just out of curiosity, why are there other fields in addition to message fields, since the issue is running as non-root ?

Reply:
@CalvoM It is just for consistency sake. On other situations, those extra fields will be used. We believe it is easier to always return the json with the same structure than to adapt it to each possible failure scenario that we might have when running the command